src/test/java/org/openstreetmap/atlas/geography/RectangleTest.java

```diff
@@ -90,6 +113,25 @@ public void testContract()
                 collapsedVertically.contract(Distance.TEN_MILES));
     }
 
+    @Test
+    public void testCoversAndCoversPartially()
+    {
+        Assert.assertTrue("Rectangle 3 fully contains rectangle 2",
+                this.rectangle3.fullyGeometricallyEncloses(this.rectangle2));
+        Assert.assertTrue("That means, it should also cover it partially",
+                this.rectangle3.overlaps(this.rectangle4));
```

Comment:
is this supposed to be 
```JAVA 
this.rectangle3.overlaps(this.rectangle2)
```
?

Reply:
Good catch! This code was re-ordered, hadn't noticed this. Fixed!